source/browseMode.py

```diff
@@ -552,6 +552,19 @@ def script_disablePassThrough(self, gesture):
 		reportPassThrough(self)
 	script_disablePassThrough.ignoreTreeInterceptorPassThrough = True
 
+	def _set_disableAutoPassThrough(self, state):
+		if (
+			state
+			and not config.conf["virtualBuffers"]["autoFocusFocusableElements"]
+			and self.passThrough

```

Comment:
Could you add a comment that describes why this check is necessary? Why would one disable auto pass through when passThrough is False?

Reply:
Honestly, there's no reason this should ever happen. I was just being cautious, since this property could theoretically be set from anywhere; e.g. an add-on.